README.md

```diff
@@ -12,6 +12,11 @@ Download Xcode and other developer tools up to 16 times faster with resume capab
 - ✨ Resume download automatically if the download failed due to any reason.    
 - ⏬ Multiple downloads supported.
 
+### How does it work?
```

Comment:
@rogerluan thanks for the PR. Can you please move it a little bit lower like after the "[how to use](https://github.com/vineetchoudhary/Downloader-for-Apple-Developers#how-to-use)" section? because not everyone looking for how it works.

Reply:
Sure thing @vineetchoudhary ! Done 💪